### ISSUE TITLE
424507D1 - KUF: Circle of Doom

### ISSUE THREAD

**Issue by CovertSlinky** [xenia-project/game-compatibility#917]
[Marketplace](https://marketplace.xbox.com/en-us/Product/KUF-Circle-of-Doom/66acd000-77fe-1000-9115-d802424507d1)

Tested on (https://github.com/xenia-project/xenia/tree/9a158963055a195c152f9057ed2f888c1f61cd6e)

# Issues:
(state-crash)
Game immediately crashes upon load whether it be load from file inside xenia itself or drag and drop game iso or default.xex, onto xenia.exe

# Log:
([xenia.zip](https://github.com/xenia-project/game-compatibility/files/1989009/xenia.zip)).

# Labels:
   (state-crash)

**Comment by Zealiia** [comment]
![captura](https://user-images.githubusercontent.com/39223550/39960458-d363c838-5623-11e8-8298-5bfedea467d0.PNG)

Hope Xenia staffs will work on it its one of the best KUF and the best game i remember on X360

**Comment by CovertSlinky** [comment]
Update
Game is currently completely playable. Only Limitation is not being able to save your game, for obvious reasons.
Please add new label Gameplay and remove state crash.
[log.zip](https://github.com/xenia-project/game-compatibility/files/4588456/log.zip)

**Comment by tucker25** [comment]
Xenia (detached/c9073e101/aug 1 2021) | 424507d1 v1.0 KUF: Circle of Doom

Game is playable without fps drops or any graphical errors during the first level (what i played so far) and you can save your game and load it even after shutting down xenia and launching it again

![Screenshot_1](https://user-images.githubusercontent.com/69175143/127953630-2cbf7ce1-ea94-4c36-a930-35325d3af273.png)

**Comment by SoaringMoon** [comment]
> Xenia (detached/c9073e101/aug 1 2021) | 424507d1 v1.0 KUF: Circle of Doom
> 
> Game is playable without fps drops or any graphical errors during the first level (what i played so far) and you can save your game and load it even after shutting down xenia and launching it again
> 
> ![Screenshot_1](https://user-images.githubusercontent.com/69175143/127953630-2cbf7ce1-ea94-4c36-a930-35325d3af273.png)

This is incorrect, the game has significant FPS drops, to single digits) any time enemies are on screen.
CPU: Ryzen 5 3600
GPU: NVIDIA 3060Ti
RAM: 64GB DDR4 2800

It happens when enemies are present, so I think it has to do with the number of agents, and not texture caching or rendering.

**Comment by Outcast-Pillow** [comment]
Is anyone still working on this title or willing to work on this? this game means the world to me it was one of the last games I was able to play with my dad so being able to play it is priceless to me. I'm not sure about the rules and such so please forgive my ignorance but I'm willing to do anything (ie. pay someone, learn to code it myself with someone's help, etc.) as long as it doesn't breach any rules. Thank you for your time in advance and I appreciate it.

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/331
